flow/activities/flowable_core.go

```diff
@@ -264,6 +266,8 @@ func syncCore[TPull connectors.CDCPullConnectorCore, TSync connectors.CDCSyncCon
 			StagingPath:            config.CdcStagingPath,
 			Script:                 config.Script,
 			TableNameSchemaMapping: tableNameSchemaMapping,
+			Env:                    config.Env,
```

Comment:
Could this have a risk of something being in env that would break the code or change the behavior?